pkg/plugin/callresource_test.go

```diff
@@ -1,4 +1,4 @@
-package plugin_test
+package plugin
```

Comment:
Why change this to the `plugin` namespace? I think it makes better tests when you separate packages so it's clear how to interact with them.

Reply:
and mocking and stuff

Reply:
If it's a separate namespace then I'm unable to import the `migrateRequest` method from the `plugin` package